CMakeLists.txt

```diff
@@ -123,9 +128,6 @@ endif()
 
 # Targets #####################################################################
 #
-if(ImpactX_PYTHON)
```

Comment:
logic in `cmake/dependencies/ABLASTR.cmake` depends on this, so we need to move it up before including that file